packages/bech32m/package.json

```diff
@@ -4,16 +4,18 @@
   "license": "(MIT OR Apache-2.0)",
   "description": "Tools for manipulating Penumbra bech32m strings",
   "type": "module",
+  "engine": {
+    "node": ">=22"
+  },
```

Comment:
engine set to latest node stable in published packages. older versions start to have trouble with typescript and package features that we use.

Reply:
specifically, older versions cause problems with importing the package - you may be running an older version internally and not experience the problem.